### ISSUE TITLE
Plugin tries to download system scope dependencies

### ISSUE THREAD

**Issue by electrum** [HubSpot/dependency-scope-maven-plugin#4]
We noticed a build failure caused by this plugin with the combination of an inaccessible remote repository and a transitive dependency on a system scope dependency. These should not be resolved / downloaded since they are special and don’t exist as real dependencies.

**Comment by jhaber** [comment]
Oof sorry about that. I believe #5 would fix this, can you try it out to confirm?

**Comment by electrum** [comment]
This fixed it for me. Thanks!

**Comment by electrum** [comment]
With `0.8`, the plugin prints this warning (or a resolution failure if the repository is unreachable):
```
[INFO] --- dependency-scope-maven-plugin:0.8:check (default) @ presto-main ---
[WARNING] The POM for com.sun:tools:jar:1.8 is missing, no dependency information available
[INFO] No test dependency scope issues found
```

**Comment by jhaber** [comment]
Thanks for filing the issue, I just released 0.10 which should show up in central soon
